Update GMT canvas

seisman opened the pull request.

seisman commented on source/basis/paper.rst:
```diff
@@ -14,7 +14,7 @@ JPG（推荐）、BMP、PPM 和 TIFF 格式。
 
     gmt begin figname pdf,png
-    gmt ...
+        gmt ...
```
我比较犹豫，这一节的示例，是按照现在的写法，给出一个框架 （即 ``gmt ...``） 还是给出具体的某个命令（比如 ``gmt basemap``）。

后者的优点在于，用户可以直接执行命令理解其输出。

core-man replied:
我比较喜欢实际例子，我觉得使用 GMT 的用户应该能举一反三，知道可以适用于其他命令。

seisman commented on source/basis/paper.rst:
```diff
@@ -8,52 +8,98 @@
+如果需要移动绘图原点，只能向右和向上移动，而不能向左或向下移动。向左或向下移动绘图原点，
+均可能使得部分图件超过画布左边距或下边界，导致图件被部分截断。
+
+TODO: 这里计划增加一张类似下面的图。
+
+.. figure:: https://docs.gmt-china.org/5.4/_images/paper-gmtplot-1.png
```
@core-man This PR is almost ready for review. The only missing part is a better figure showing the position of the plot origin on the full-size GMT paper.

I'll work on the image if you think the other parts are all OK.

seisman commented:
![map](https://user-images.githubusercontent.com/3974108/129662085-62875cc5-05b4-4123-a431-687cddfd89d9.png)

I added this figure to explain the relative positioning of plot origin on a "full-size" paper.

core-man commented on source/basis/paper.rst:
```diff
@@ -1,59 +1,105 @@
-GMT 中默认使用宽 11.55 米、高 11.55 米的画布，绘图原点位于画布左下角。
-在最终生成图片文件时会自动将未使用的白色区域裁减掉，因而用户无需关心画布大小。
+GMT 中默认使用宽 11.6 米、高 11.6 米的画布，在最终生成图片文件时会自动将未使用的白色区域裁减掉，
+因而用户无需关心画布大小。
 
 若需要生成特定尺寸的图片，而不裁剪周边的白色区域，可以修改 GMT 参数
 :term:`PS_MEDIA` 来指定纸张尺寸::
```
```suggestion
因而用户无需关心画布大小。若需要生成特定尺寸的图片，而不裁剪周边的白色区域，
可以修改 GMT 参数 :term:`PS_MEDIA` 来指定纸张尺寸::
```

core-man commented on source/basis/paper.rst:
```diff
@@ -1,59 +1,105 @@
 图片格式
 --------
 
 GMT 支持多种矢量图片格式和位图图片格式。
```
```suggestion
GMT 支持多种矢量图片格式和位图图片格式：
```

core-man commented on source/basis/paper.rst:
```diff
@@ -1,59 +1,105 @@
 
-矢量图片格式包括 PDF（推荐使用）、PS 和 EPS 格式；位图图片格式包括 PNG（推荐）、
-JPG（推荐）、BMP、PPM 和 TIFF 格式。
+矢量图片格式包括 PDF [**推荐**]、PS 和 EPS 格式；位图图片格式包括 PNG [**推荐**]、
+JPG [**推荐**]、BMP、PPM 和 TIFF 格式。
```
```suggestion
- 矢量图片格式包括 PDF [**推荐**]、PS 和 EPS 格式；
- 位图图片格式包括 PNG [**推荐**]、JPG [**推荐**]、BMP、PPM 和 TIFF 格式。
```

seisman commented:
@core-man I renamed the file name from `paper.rst` to `canvas.rst` because "canvas" is the correct name. The PR now is a little difficult to review due to the file name change, so you have to read the whole section again.

The script used to generate the figure is also added to this repository, but the script is neither executed by the gmtplot directive, nor shown in the documentation. It's included in the repo to make sure that we can reproduce the image in the future.

seisman commented:
Ping @core-man for review.

wangliang1989 commented on source/basis/canvas.rst:
```diff
@@ -0,0 +1,103 @@
+绘图原点
+--------
+
+GMT 在绘图时，默认的绘图原点位于画布（11.6 米宽、11.6 米高）的左下角 (12.7 cm, 12.7 cm) 处
```
初学者可能不理解什么是绘图原点。应该先陈述绘图原点的定义，再说它在哪

seisman replied:
Fixed in a18acff4.

wangliang1989 commented on source/basis/canvas.rst:
```diff
@@ -0,0 +1,103 @@
+绘图原点
+--------
+
+GMT 在绘图时，默认的绘图原点（即底图边框左下角）位于画布（11.6 米宽、11.6 米高）的
```
**前面解释底图了吗？**
```suggestion
绘图原点是指底图的左下角。默认的绘图原点（即底图边框左下角）位于画布（11.6 米宽、11.6 米高）的
```

seisman replied:
默认读者知道底图

wangliang1989 replied:
我觉得他很可能不知道何为底图
